frontend/src/constants/breadcrumbs.ts

```diff
@@ -26,3 +27,4 @@ export const NEWSLETTER_UNSUBSCRIBE_CRUMBS: BreadcrumbList = [
   NEWSLETTER,
   NEWSLETTER_UNSUBSCRIBE,
 ];
+export const SEARCH_CRUMBS: BreadcrumbList = [HOME, SEARCH];
```

Comment:
nit: maybe a TODO here also if this is setup but not yet working?

Reply:
This file is working. There's a TODO comment in the page calling it.